src/tests/cli_tests.py

```diff
@@ -777,6 +777,21 @@ def rnp_misc_operations():
     run_test(rnp_encryption_no_mdc)
     run_test(rnp_encryption_s2k)
 
+def rnp_armour():
+    src_beg,dst_beg = reg_workfiles('beg','.src','.dst')
```

Comment:
Maybe add some more test cases with different message sizes and headers?
Actually, armouring itself will be tested within encryption/decryption and signing/verification. But this could test for validity of headers and so on.

Reply:
yeah, I'll work on tests. for some reason they are broken on CI but working localy

Reply:
It fails in CI because rnp considers it a fatal error when the pubring can't be read [here](https://github.com/riboseinc/rnp/blob/master/src/rnp/rnp.c#L744). We don't need keys to enarmor so I would say that's a bug.

(Also since your `run_proc` invocations don't include `'--homedir', RNPDIR`, this will mkdir ~/.rnp which isn't ideal for a test.)

Reply:
Aha! Thnx. Indeed that's a bug - we don't need to read keys in this case (or call keystore for anything else).